### DISCUSSION TITLE
Changing the opacity of the static backdrop in HxModal

### DISCUSSION THREAD

**Discussion by BEX-IT-Johannes** [havit/Havit.Blazor#751] [Q&A] [answered]
Hi guys, this is probably an easy one and I'm probably just tired after a long days work, but how do I make the static backdrop less transparent? Right now the modal dialog blends in a bit too much with the underlying UI which makes it a bit "cluttered", so I'd like to dim down the background a bit.

**Comment by crdo** [comment, marked as answer]
There is a CSS var `--bs-backdrop-opacity` that is scoped to `.modal-backdrop.show` CSS class.
The default is set by Bootstrap to `.5`.

**Reply by BEX-IT-Johannes** [reply]
Ah, thanks for the help. :)
